website/docs/mutation-examples/seccomp.md

```diff
@@ -7,23 +7,30 @@ title: seccomp
 
 ### Usage
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper-library/master/mutation/pod-security-policy/seccomp/samples/mutation.yaml
```

Comment:
Hmm, looks like we are missing `mutation-annotation`.

@nilekhc Looks like the website generator currently will only retain the final mutation sample per directory? https://github.com/open-policy-agent/gatekeeper-library/blob/master/scripts/website/generate.go#L226